cmd/agent.go

```diff
@@ -345,6 +348,33 @@ func (ar *AgentRunner) runInstance() error {
 
 		logger.Debug("Running plugin", "source", source)
 
+		uuidSeedData := map[string]string{
+			// Repeatably identify this plugin / policy combination.
+			// Rerunning this plugin / policy on the same agent with the same config should generate the same UUID.
+			"plugin-name":    pluginName,
+			"plugin-version": "1.0.0",
```

Comment:
Any reason we can't identify what plugin version we have currently? Not urgent, can be done later. Might be worth marking some kind of TODO if that's the plan.

In theory we can assume for now though that we always just get this from this config. Future iterations could then do things like `latest`, `~>` from Terraform type of things